nb/api/in.md

```diff
@@ -0,0 +1,28 @@
+### in()
+Produces the interior of an item.
```

Comment:
I think that people might read this as it gives the volume contained in the solid that is represented by the item. I know what you mean, and I'm not sure how else to say it, but I think first time users might not get it. Maybe "Produces the interior of an items tree"? Or something?

Reply:
Hmm, good point.

Reply:
How about "Produces the shape structure inside the item." ?

Reply:
I like that. I think that the examples are really what makes it clear